pkg/install/container.go

```diff
@@ -0,0 +1,147 @@
+package install
+
+import (
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/datawire/ambassador/pkg/kates"
+)
+
+const envPrefix = "TEL_APP_"
+
+func AgentContainer(name, imageName string, appContainer *v1.Container, port v1.ContainerPort, appPort int, managerNs string) v1.Container {
```

Comment:
That's a lot of args.  Maybe it should take a single struct?

At least wrap this to be 1 arg per line.